Config/BuildSettings.swift

```diff
@@ -399,6 +399,13 @@ final class BuildSettings: NSObject {
     // MARK: - Polls
     
     static let pollsEnabled = true
+    static var pollsHistoryEnabled: Bool {
+        #if DEBUG
+        true
+        #else
+        false
+        #endif
```

Comment:
I guess that was because of my laziness in switching the flag every time I work on this. 😆
Btw your suggestion looks ok to me!
https://github.com/vector-im/element-ios/pull/7267/commits/e1b04c10361642e699484fa33a1773ac8de549d9